tests/test_encoding/test_decision_tree_encoder.py

```diff
@@ -109,3 +116,17 @@ def test_variables_cast_as_category(df_enc_category_dtypes):
     transf_df["var_B"] = [0.2] * 10 + [0.4] * 10  # Tree: var_B <= 0.5 -> 0.2 else 0.4
     pd.testing.assert_frame_equal(X, transf_df[["var_A", "var_B"]], check_dtype=False)
     assert X["var_A"].dtypes == float
+
+
+def test_error_is_regression_true_and_target_variable_binary(df_enc):
+    with pytest.raises(ValueError):
+        encoder = DecisionTreeEncoder(regression=True)
+        encoder.fit(df_enc[["var_A", "var_B"]], df_enc["target"])
+
+
+def test_error_is_regression_false_and_target_variable_continuous(df_enc_numeric):
```

Comment:
same here, can we change test name to:

`test_error_when_regression_is_false_and_target_is_continuous`